npm/qsharp/ux/circuit-vis/events.ts

```diff
@@ -450,9 +455,12 @@ class CircuitEvents {
           ev.stopPropagation(),
         );
         dropzone.addEventListener("click", () => {
-          if (this.selectedOperation != null) {
-            const successful = addControl(this.selectedOperation, wireIndex);
-            this.selectedOperation = null;
+          if (
+            this.selectedComponent != null &&
+            this.selectedComponent.type === "Operation"
+          ) {
+            const successful = addControl(this.selectedComponent, wireIndex);
+            this.selectedComponent = null;
```

Comment:
This looks like an improvement to me, because you can't add controls to a measurement?